bot3.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * https://github.com/cvzi/telegram-bot-cloudflare
+ */
+
+const TOKEN = ENV_BOT_TOKEN // Get it from @BotFather https://core.telegram.org/bots#6-botfather
+const WEBHOOK = '/endpoint'
+const SECRET = ENV_BOT_SECRET // A-Z, a-z, 0-9, _ and -
+
+/**
+ * Wait for requests to the worker
+ */
+addEventListener('fetch', event => {
+  const url = new URL(event.request.url)
+  if (url.pathname === WEBHOOK) {
+    event.respondWith(handleWebhook(event))
+  } else if (url.pathname === '/registerWebhook') {
+    event.respondWith(registerWebhook(event, url, WEBHOOK, SECRET))
+  } else if (url.pathname === '/unRegisterWebhook') {
+    event.respondWith(unRegisterWebhook(event))
+  } else {
+    event.respondWith(new Response('No handler for this request'))
+  }
+})
+
+/**
+ * Handle requests to WEBHOOK
+ * https://core.telegram.org/bots/api#update
+ */
+async function handleWebhook (event) {
+  // Check secret
+  if (event.request.headers.get('X-Telegram-Bot-Api-Secret-Token') !== SECRET) {
+    return new Response('Unauthorized', { status: 403 })
+  }
+
+  // Read request body synchronously
+  const update = await event.request.json()
+  // Deal with response asynchronously
+  event.waitUntil(onUpdate(update))
+
+  return new Response('Ok')
+}
+
+/**
+ * Handle incoming Update
+ * https://core.telegram.org/bots/api#update
+ */
+async function onUpdate (update) {
+  if ('message' in update) {
+    await onMessage(update.message)
+  }else if ('inline_query' in update){
+    await onInlineQuery(update.inline_query)
+  }
+}
+
+/**
+ * Handle incoming Message
+ * https://core.telegram.org/bots/api#message
+ */
+function onMessage (message) {
+  return sendPlainText(message.chat.id, 'This is an inline bot')
+
+}
+
+/**
+ * Send plain text message
+ * https://core.telegram.org/bots/api#sendmessage
+ */
+async function sendPlainText (chatId, text) {
+  return (await fetch(apiUrl('sendMessage', {
+    chat_id: chatId,
+    text
+  }))).json()
+}
+
+/**
+ * Handle incoming query
+ * https://core.telegram.org/bots/api#InlineQuery
+ * This will reply with a voice message but can be changed in type
+ * The input file is defined in the environment variables.
+ */
+async function onInlineQuery (inline_query) {
+var results = [];
+var res ="";
+var texy_json_input_files =""
```

Comment:
This variable is unused. Can it be removed or is it a mistake and it should be used somewhere?

Reply:
Thanks, it remained from when I was testing it. Now it is removed